requirements.txt

```diff
@@ -4,7 +4,7 @@ numba
 torch>=1.1
 tensorboardX
 easydict
-pyyaml
+pyyaml==5.4.1
```

Comment:
May I know why need to fix the pyyaml version?

Reply:
Based on source, it use yaml.load() function for loading yaml scripts, but it is deprecated in latest version and causing error : "load() missing 1 required positional argument 'loader'". 

So, it might need to fix the pyyaml version supporting yaml.load() function.
(Another option i found is replacing it to yaml.full_load() in latest version of pyyaml)